frontends/article/components/ArticleActions/article-action-icons.tsx

```diff
@@ -56,14 +63,14 @@ export const ArticleActionIcons: FC<IArticleActionsIconsProps> = ({
     dispatch(toggleCommentsSidebar());
   };
   const likeIcon = useMemo((): JSX.Element => {
-    if (isLikedOrDisliked === 0) {
+    if (isLikedOrDisliked === false) {
```

Comment:
check-like-dislike api: 
returns false if article is disliked
returns true if article is liked
returns null/undefined if article is neither liked, nor disliked